### PR TITLE
Several cosmetic changes

### PR DESCRIPTION
**Pull request by matiasgarcia91**
### :pushpin: References

* **Issue:** Closes several issues of #7 

### :memo: Implementation

-

### :art: Screenshots
image

### :fire: Is there anything the reviewer should know to test it?
I was going to add a help text but since I'm not completely familiar with the use-case of this app it would probably need changing. We need to define it and I'll update it.

### :bookmark_tabs: Others

- Both in this app and in vaccination we are having a bit of an issue with the spacing on the top of the objectsTable, is it worth it to modify it in the d2-ui library?

### CODE REVIEW

File: src/components/instance-form-builder/InstanceFormBuilder.jsx [hunk 1]
```diff
@@ -68,7 +68,11 @@ class InstanceFormBuilder extends React.Component {
                     contents={i18n.t("All your changes will be lost. Are you sure?")}
                 />
 
-                <FormHeading title={title} onBackClick={this.cancelSave} />
+                <PageHeader
+                    title={title}
+                    onBackClick={this.cancelSave}
+                    helpText={i18n.t("Help text")}
```
Comment (SferaDev) [inline src/components/instance-form-builder/InstanceFormBuilder.jsx]: If we don't have a proper help text now we could just remove it for now and define it later on. @adrianq Might have more insight about this.

**Comment by matiasgarcia91** [comment]
@SferaDev thanks for the details! You worked on the ObjectsTable in d2-ui-components, what do you think about this?

> Both in this app and in vaccination we are having a bit of an issue with the spacing on the top of the objectsTable, is it worth it to modify it in the d2-ui library?

File: src/components/instance-form-builder/InstanceFormBuilder.jsx (hunk 1 above)
Comment (matiasgarcia91) [reply src/components/instance-form-builder/InstanceFormBuilder.jsx]: We'll wait a bit on this, if not I'll try and write up something coherent

File: src/components/instance-configurator/InstanceConfigurator.jsx [hunk 2]
```diff
@@ -79,24 +80,31 @@ class InstanceConfigurator extends React.Component {
...
-                    pageSize={10}
-                    initialSorting={this.initialSorting}
-                    actions={this.actions}
-                    onCreate={this.onCreate}
-                    list={Instance.list}
-                />
-            </div>
+            <React.Fragment>
+                <PageHeader title={i18n.t("Instances")} onBackClick={this.backHome} />
+                <div style={{ marginTop: -10 }}>
```
Comment (tokland) [inline src/components/instance-configurator/InstanceConfigurator.jsx]: As @SferaDev already noted, let's use `withStyle`

Comment (matiasgarcia91) [reply src/components/instance-configurator/InstanceConfigurator.jsx]: I left this on hold while waiting for the ObjectsTable comment to be answered, I'll change it and we'll remove it when we update d2-ui-components.

File: src/components/instance-form-builder/InstanceFormBuilder.jsx (hunk 1 above)
Comment (adrianq) [reply src/components/instance-form-builder/InstanceFormBuilder.jsx]: This is fine for now. We can always revisit it at a later stage. I just want to show the client what we can potentially have and if it makes sense for them

**Pull request merged**